frontend/src/components/DataCard/index.jsx

```diff
@@ -32,22 +35,43 @@ class DataCard extends React.Component {
                         <h2>Card {cards[0].id + 1}</h2>
                         <p>{cards[0].text["text"]}</p>
                         <ButtonToolbar className="btn-toolbar pull-right">
-                            {labels.map(opt => (
-                                <Button
-                                    onClick={() =>
+                            {labels.length > 5 ?
+                                <Select
+                                    className="align-items-center flex py-1 px-2"
+                                    dropdownHandle={false}
+                                    labelField="name"
+                                    onChange={(value) =>
                                         annotateCard(
                                             cards[0],
-                                            opt["pk"],
+                                            value[0]["pk"],
                                             cards.length,
                                             ADMIN
                                         )
                                     }
-                                    variant="primary"
-                                    key={`deck-button-${opt["name"]}`}
-                                >
-                                    {opt["name"]}
-                                </Button>
-                            ))}
+                                    options={labelsOptions}
+                                    placeholder="Select label..."
+                                    searchBy="name"
+                                    sortBy="name"
+                                    style={{ minWidth: '200px' }}
+                                />
+                            :
```

Comment:
So when I run this, I get a linting error for this line. Did the pre-commit linting run properly when you committed this file? You can see the error if you run this `docker-compose -f envs/dev/docker-compose.yml run --rm smart_frontend yarn lint`

Reply:
Good catch. I'm not sure why mine did not run, but I think that file should be fixed now